README.md

```diff
@@ -147,6 +147,16 @@ return (
 )
 ```
 
+You can provide a `container` prop that accepts an HTML element that is forwarded to Radix UI's Dialog Portal component to specify which element the Dialog should portal into (defaults to `body`). See the [Radix Documentation](https://www.radix-ui.com/docs/primitives/components/dialog#portal) for more information.
+
+```tsx
+const element = document.querySelector("#container")
```

Comment:
Should the example use `useRef` instead? You would likely never use `.querySelector` inside of a render function like this code snippet suggests.

Reply:
This makes clear that it's an HTML element (per radix' docs) not a React component, and the element might be in a completely different part of your document so a `useRef` might not be practical so that's why I went for this, but happy to change it.

Reply:
changed, let me know if new notation is ok.